lib/multer-error.js

```diff
@@ -0,0 +1,13 @@
+var util = require('util')
+
+function MulterError (message, code, field) {
+  Error.captureStackTrace(this, this.constructor)
+  this.name = this.constructor.name
+  this.message = message
+  this.code = code
+  this.field = field
```

Comment:
Could you add back the `if` statement? As it is now, `'field' in err` will evaluate to true, even for errors without fields.

Reply:
Ugh, I am sorry. I didn't notice that.